icu4j/main/core/src/test/java/com/ibm/icu/dev/test/message2/ParserSmokeTest.java

```diff
@@ -29,16 +29,5 @@ public void testNullInput() throws Exception {
         MFParser.parse(null);
     }
 
-    @Test
```

Comment:
There is not much / anything left if we remove this.

So the same comment that I had for deleted files applies: do we really want to delete extra tests?
I think that the more tests we have the better.
Even some don't come from the official CLDR spec.

We can separate these tests in a different folder for cleanliness.
But not delete them, unless they are redundant.

Reply:
See https://github.com/unicode-org/icu/pull/3092#discussion_r1712036379